src/sentry/static/sentry/app/utils/discover/discoverQuery.tsx

```diff
@@ -197,20 +198,24 @@ class DiscoverQuery extends React.Component<Props, State> {
     apiPayload: EventQuery & LocationQuery & TrendsQuery,
     location: Location
   ) => {
-    const {trendChangeType, eventView} = this.props;
+    const {trendChangeType, eventView, isTrendsProject} = this.props;
     if (trendChangeType) {
       delete apiPayload.cursor;
-      if (
-        trendChangeType === TrendChangeType.IMPROVED &&
-        location?.query?.improvedCursor
-      ) {
-        apiPayload.cursor = location?.query?.improvedCursor;
-      } else if (
-        trendChangeType === TrendChangeType.REGRESSION &&
-        location?.query?.regressionCursor
-      ) {
-        apiPayload.cursor = location?.query?.regressionCursor;
+
+      if (!isTrendsProject) {
```

Comment:
Should this be negated? Feels like it should just be `if (isTrendsProject)`

Reply:
Yes, project trends shouldn't have cursor added to their payload.